ERP/routes/web.php

```diff
@@ -39,14 +39,19 @@
 Route::post('/inventory', [MaterialController::class, 'createMaterial'])
 ->name('create.material');
 
-Route::get('/jobs', function(){
-    return view ('jobs');
-})
-->middleware('auth')
-->name("inventory");
-
-Route::post('/jobs', [JobController::class, 'createJob'])
-->name('create.job');
+// Route::get('/jobs', function(){
+//     return view ('jobs');
+// })
+// ->middleware('auth')
+// ->name("inventory");
+
+// Route::post('/jobs', [JobController::class, 'createJob'])
+// ->name('create.job');
+
+Route::get ('/jobs', [JobController::class, 'goToJobManagement']);
+Route::get('/create-job', [JobController::class, 'goToCreateJob']);
+Route::post('/create-job', [JobController::class, 'createJob'])
+    ->name('create.job');
```

Comment:
Add **->middleware('auth')** to your routes
Maybe add -**>name()** [but this is optional, its useful for when you want to use the route() method]